src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -80,6 +80,16 @@ namespace OpenMS
       return db_;
     }
 
+    inline static ModificationsDB * getCustomInstance(OpenMS::String unimod_file, OpenMS::String psimod_file)
```

Comment:
For TOPP tools we could also add the path to the different xml files as advanced parameters in TOPPBase. If we then just have to make sure that the first thing we do is instanciate the dbs one after the other in the correct order.